CMakeLists.txt

```diff
@@ -166,6 +166,7 @@ option(NGRAPH_LIB_VERSIONING_ENABLE "Enable shared library versioning" FALSE)
 option(NGRAPH_PYTHON_BUILD_ENABLE "Enable build nGraph python package wheel" FALSE)
 option(NGRAPH_PLAIDML_ENABLE "Enable the PlaidML backend" ${PLAIDML_FOUND})
 option(NGRAPH_DISTRIBUTED_ENABLE "Enable distributed training using MLSL/OpenMPI" OFF)
+option(NGRAPH_DISABLE_FAST_MATH "Disable fast math" OFF)
```

Comment:
How about `NGRAPH_FAST_MATH_ENABLE` with the default of ON. That way it follows the naming convention of the other flags.

Reply:
would this say we are currently using eigen fast math already?

Reply:
yes, we are enabling by default.